.github/workflows/jepsen-cluster.yml

```diff
@@ -0,0 +1,65 @@
+name: jepsen-cluster
+
+on:
```

Comment:
And here - in both cases run them via cron, but rather using different periods of time (shorter and longer depending on test)